### PR TITLE
Add Custom Dictionaries

### PR DESCRIPTION
**Pull request by sbason**
Added British English as first non-US dictionary.

### CODE REVIEW

**Comment by aotaduy** [comment]
Thanks Sam. Will test and merge.

File: test/spell-checker.js [hunk 1]
```diff
@@ -78,6 +78,13 @@ eslintTester.addRuleTest('rules/spell-checker', {
             errors: [
                 { message: 'You have a misspelled word: tsih on String'},
                 { message: 'You have a misspelled word: srting on String'}]
+        },
+        {
+            code: 'var a = 1 // color is a comment normalize dict',
+            args:[2, {lang: 'en_GB', skipWords: ['dict']}],
+            errors: [
+                { message: 'You have a misspelled word: color on Comment'},
+                { message: 'You have a misspelled word: normalize on Comment'}]
```
Comment (aotaduy) [inline test/spell-checker.js]: I tested locally and normalize is not failing in my local test. Could you please correct and find another word not present in the US dict ?
And please add another test case for the British happy path with British words and the test not failing. 
Something like 
````
 code: 'var cheque = 1 // cheque is in colour tsih is a comment dict',
             args:[2, {lang: 'en_GB', skipWords: ['dict']}],
             errors: [
                 { message: 'You have a misspelled word: tsih on Comment'}]
```
thanks for the PR!

Comment (sbason) [inline test/spell-checker.js]: Sure will do - this is clearly why I need a spell checker :) 

Also is there a way for me to run the tests locally?

Comment (aotaduy) [inline test/spell-checker.js]: Yes! you have to install gulp
```
npm install -g gulp
```
and in the project folder for the first time:
```
npm install
```
then to run tests:
```
gulp test
```

You can even check for coverage in the ```coverage```folder

Comment (sbason) [inline test/spell-checker.js]: Ah it was Gulp I was missing. Should all be passing now.

**Comment by aotaduy** [comment]
Thanks ! will merge and publish

**Pull request merged**